### DISCUSSION TITLE
US 4G LTE Carriers

### DISCUSSION THREAD

**Discussion by blu31c33** [tesla-android/issue-tracker#14] [Q&A] [answered]
can anyone using this in the US recommend a good 4g lte carrier that works with the alcatel usb stick?  i checked with t-mobile and their pricing is very high just for data.  i think this usb stick also works with at&t (which will also probably be high).  i was hoping to pay closer to the 9.99 tesla charges.  any suggestions?

**Comment by blu31c33** [comment, marked as answer]
i went with at&t.  works great.  way more expensive than the 10 bucks from tesla.

**Reply by focher** [reply]
You could try one of the alternatives like Mint or Tracfone.
